src/main/java/oss/fosslight/api/controller/v1/ApiOssController.java

```diff
@@ -9,19 +9,17 @@
 import java.util.List;
 import java.util.Map;
 
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestHeader;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Turn off the wildcard setting in your import statement.

https://velog.io/@eora21/Intellij%EC%97%90%EC%84%9C-%EC%99%80%EC%9D%BC%EB%93%9C%EC%B9%B4%EB%93%9C-%EC%9E%84%ED%8F%AC%ED%8A%B8-%EC%84%A4%EC%A0%95-%EB%B3%80%EA%B2%BD%ED%95%98%EA%B8%B0